src/Event/Http/Psr7Bridge.php

```diff
@@ -166,4 +168,19 @@ private static function parseKeyAndInsertValueInArray(array &$array, string $key
 
         $pointer = $value;
     }
+
+    /**
+     * Cleanup previously uploaded files.
+     */
+    public static function cleanupUploadedFiles(): void
+    {
+        $tmpFiles = glob(sys_get_temp_dir() . '/' . self::UPLOADED_FILES_PREFIX . '*');
+        if ($tmpFiles !== false) {
+            foreach ($tmpFiles as $file) {
+                if (is_file($file)) {
+                    unlink($file);
```

Comment:
This could fail if the previous request didn't cleanup the open handles properly.

Reply:
In which case is there anything we can do? I don't want to turn this into an error and break stuff

Reply:
Maybe we need to silence it?

Reply:
🤦 I forgot about custom error handlers that might turn the warning into exceptions! (I was focused on the `bool` return). The joys of warnings.